### PR TITLE
chore(color-area): removed deprecated label from color-area

### PR DESCRIPTION
**Pull request by blunteshwar**
## Description

removed deprecated `label` attribute from color-area for SWC 1.0 release

## Related issue(s)
#1802 

-

## Types of changes

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)
-   [x] Chore (minor updates related to the tooling or maintenance of the repository, does not impact compiled assets)

## Checklist

-   [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
-   [x] My code follows the code style of this project.
-   [x] If my change required a change to the documentation, I have updated the documentation in this pull request.
-   [x] I have read the **[CONTRIBUTING](<(https://github.com/adobe/spectrum-web-components/blob/main/CONTRIBUTING.md)>)** document.
-   [x] I have added tests to cover my changes.
-   [x] All new and existing tests passed.
-   [x] I have reviewed at the Accessibility Practices for this feature, see: [Aria Practices](https://www.w3.org/TR/wai-aria-practices/)

## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.

### CODE REVIEW

**Review by Rajdeepc — APPROVED** [review]

File: packages/color-area/test/color-area.test.ts [hunk 1]
```diff
@@ -131,31 +123,10 @@ describe('ColorArea', () => {
             '75%, luminosity, 67%, saturation'
         );
     });
-    it('overrides both X and Y labels with a provided "label" attribute', async () => {
```
Comment (Westbrook) [inline packages/color-area/test/color-area.test.ts:134]: We should replace this with a test using `label-x` and `label-y` to customize the label.

Comment (blunteshwar) [reply packages/color-area/test/color-area.test.ts:134]: Done!

**Review by Westbrook — APPROVED** [review]
LGTM! Have yall discussed whether you'd make your removals on a rolling basis or as a big bang 1.0?

**Comment by Rajdeepc** [comment]
@blunteshwar Pull this up to `main` and we can merge this!

**Comment by blunteshwar** [comment]
> @blunteshwar Pull this up to `main` and we can merge this!

Done!

**Review by Rajdeepc — APPROVED** [review]

**Pull request merged**